### PR TITLE
Allow for unsubscription to the weekly mail

### PR DESCRIPTION
**Pull request by edvardvb**
Would something like this work, @eirsyl? Don't really know how to test this :confused:

### CODE REVIEW

**Comment by eirsyl** [comment]
works for now i guess 😄

**Comment by edvardvb** [comment]
Merge so we can test in prod? :innocent:

**Comment by eirsyl** [comment]
Maybe add a test that creates a notification setting for a user and makes sure lookup_recipients dont return the email?

File: lego/apps/restricted/tests/test_models.py [hunk 1]
```diff
@@ -37,3 +40,31 @@ def test_invalid_lookup(self):
...
+                                           enabled=True,
+                                           channels=[PUSH])
+        NotificationSetting.objects.create(user=subscribed,
+                                           notification_type=WEEKLY_MAIL,
+                                           enabled=True,
+                                           channels=[EMAIL])
+
+        recipients = restricted_mail.lookup_recipients()
+
+        self.assertCountEqual(recipients, [
```
Comment (eirsyl) [inline lego/apps/restricted/tests/test_models.py]: Assert list equal, not count.

**Pull request merged**